README.rst

```diff
@@ -81,20 +81,20 @@ physical constants and commonly used constants.
 
 The typical usage is the following:
 
-.. code-block:: python
+.. code-block:: pycon
 
     >>> from hepunits.constants import c_light
-    >>> from hepunits.units     import picosecond, micrometer
-    >>> tau_Bs = 1.5 * picosecond    # a particle lifetime, say the Bs meson's
-    >>> ctau_Bs = c_light * tau_Bs   # ctau of the particle, ~450 microns
-    >>> print ctau_Bs                # result in HEP units, so mm
+    >>> from hepunits.units import picosecond, micrometer
+    >>> tau_Bs = 1.5 * picosecond  # a particle lifetime, say the Bs meson's
+    >>> ctau_Bs = c_light * tau_Bs  # ctau of the particle, ~450 microns
+    >>> print(ctau_Bs)  # result in HEP units, so mm
     0.449688687
-    >>> print ctau_Bs / micrometer   # result in micrometers
+    >>> print(ctau_Bs / micrometer)  # result in micrometers
```

Comment:
Wow, how that went unnoticed for so long since moving away from Python 2!? Good catch!